### PR TITLE
Saving kickstart server changes just in case

### PR DESCRIPTION
**Pull request by broble**

### CODE REVIEW

**Comment by broble** [comment]
temp closing

**Comment by broble** [comment]
re-opening since it sounds like a few of these are needed

File: CentOS_6_Teeth_post.sh [hunk 1]
```diff
@@ -58,6 +58,15 @@ EOF
...
 #ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules
+cat > /etc/udev/rules.d/70-persistent-net.rules <<'EOF'
+#OnMetal v1
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.1", NAME="eth1"
+
+#OnMetal v2
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.1", NAME="eth1"
+EOF
```
Comment (naterh) [inline CentOS_6_Teeth_post.sh]: These changes are needed

File: CentOS_7_Teeth.cfg [hunk 2]
```diff
@@ -136,7 +136,15 @@ EOF
...
-echo -n > /etc/udev/rules.d/70-persistent-net.rules
+cat > /etc/udev/rules.d/70-persistent-net.rules <<'EOF'
+#OnMetal v1
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.1", NAME="eth1"
+
+#OnMetal v2
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.1", NAME="eth1"
+EOF
```
Comment (naterh) [inline CentOS_7_Teeth.cfg]: These changes are needed

File: Debian_8_Teeth_post.sh [hunk 3]
```diff
@@ -70,7 +70,16 @@ mounts:
...
+#OnMetal v1
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.1", NAME="eth1"
+
+#OnMetal v2
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.1", NAME="eth1"
+EOF
+
 echo -n > /lib/udev/rules.d/75-persistent-net-generator.rules
```
Comment (naterh) [inline Debian_8_Teeth_post.sh]: These changes are needed

File: Debian_Testing_Teeth_post.sh [hunk 4]
```diff
@@ -88,7 +88,16 @@ cloud_config_modules:
...
+#OnMetal v1
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:08:00.1", NAME="eth1"
+
+#OnMetal v2
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.0", NAME="eth0"
+SUBSYSTEM=="net", ACTION=="add", KERNELS=="0000:03:00.1", NAME="eth1"
+EOF
+
 echo -n > /lib/udev/rules.d/75-persistent-net-generator.rules
```
Comment (naterh) [inline Debian_Testing_Teeth_post.sh]: These changes are needed

File: Debian_Unstable_Teeth_post.sh [hunk 5]
```diff
@@ -91,6 +91,39 @@ EOF
...
+Name=eth0
+EOF
+cat > /etc/systemd/network/13-internet.link <<'EOF'
+[Match]
+Path=pci-0000:08:00.1-*
+
+[Link]
+Name=eth1
+EOF
+
```
Comment (naterh) [inline Debian_Unstable_Teeth_post.sh]: These changes are needed

File: Debian_Unstable_Teeth_post.sh [hunk 6]
```diff
@@ -127,7 +160,7 @@ sed -i 's/#GRUB_DISABLE_LINUX_UUID.*/GRUB_DISABLE_LINUX_UUID="true"/g' /etc/defa
 #sed -i 's/#GRUB_TERMINAL=console/GRUB_TERMINAL=console/g' /etc/default/grub
 sed -i 's/#GRUB_TERMINAL=console/GRUB_TERMINAL=/g' /etc/default/grub
 #sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT.*/GRUB_CMDLINE_LINUX_DEFAULT="console=ttyS4,115200n8 8250.nr_uarts=5 splash quiet"/g' /etc/default/grub
-sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT.*/GRUB_CMDLINE_LINUX_DEFAULT="rd.fstab=no acpi=noirq noapic 8250.nr_uarts=5 quiet"/g' /etc/default/grub
+sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT.*/GRUB_CMDLINE_LINUX_DEFAULT="rd.fstab=no acpi=ht noapic 8250.nr_uarts=5 quiet"/g' /etc/default/grub
```
Comment (naterh) [inline Debian_Unstable_Teeth_post.sh]: These changes are needed

File: Fedora_22_Teeth.cfg [hunk 7]
```diff
@@ -152,6 +152,41 @@ echo "dns=none" >> /etc/NetworkManager/NetworkManager.conf
...
+[Link]
+Name=eth0
+EOF
+cat > /etc/systemd/network/13-eth.link <<'EOF'
+[Match]
+Path=pci-0000:08:00.1-*
+
+[Link]
+Name=eth1
+EOF
```
Comment (naterh) [inline Fedora_22_Teeth.cfg]: These changes are needed

File: Fedora_22_Teeth.cfg [hunk 8]
```diff
@@ -266,6 +301,10 @@ chmod +x /etc/rc.d/rc.local
 systemctl enable rc-local.service
 systemctl disable NetworkManager.service
 
+# disable acpid soft-off
+chkconfig --del acpid 
+systemctl disable acpid.service
+
```
Comment (naterh) [inline Fedora_22_Teeth.cfg]: These changes are needed

File: Fedora_22_Teeth.cfg [hunk 9]
```diff
@@ -279,7 +318,7 @@ default linux
 
 LABEL linux
       KERNEL /boot/vmlinuz-$version
-      APPEND root=LABEL=root rd.fstab=no acpi=noirq noapic modprobe.blacklist=mei_me rd.auto=1
+      APPEND root=LABEL=root rd.fstab=no biosdevname=0 net.ifnames=0 modprobe.blacklist=mei_me rd.auto=1
```
Comment (naterh) [inline Fedora_22_Teeth.cfg]: These changes are needed

File: Fedora_23_Teeth.cfg [hunk 10]
```diff
@@ -144,6 +144,41 @@ echo "dns=none" >> /etc/NetworkManager/NetworkManager.conf
...
+[Link]
+Name=eth0
+EOF
+cat > /etc/systemd/network/13-eth.link <<'EOF'
+[Match]
+Path=pci-0000:08:00.1-*
+
+[Link]
+Name=eth1
+EOF
```
Comment (naterh) [inline Fedora_23_Teeth.cfg]: These changes are needed

File: Fedora_23_Teeth.cfg [hunk 11]
```diff
@@ -261,6 +296,10 @@ chmod +x /etc/rc.d/rc.local
 systemctl enable rc-local.service
 systemctl disable NetworkManager.service
 
+# disable acpi soft-off
+chkconfig --del acpid
+systemctl disable acpid.service
+
```
Comment (naterh) [inline Fedora_23_Teeth.cfg]: These changes are needed

File: Fedora_23_Teeth.cfg [hunk 12]
```diff
@@ -274,7 +313,7 @@ default linux
 
 LABEL linux
       KERNEL /boot/vmlinuz-$version
-      APPEND root=LABEL=root net.ifnames=0 biosdevname=0 rd.fstab=no acpi=noirq noapic modprobe.blacklist=mei_me rd.auto=1
+      APPEND root=LABEL=root net.ifnames=0 biosdevname=0 rd.fstab=no modprobe.blacklist=mei_me rd.auto=1
```
Comment (naterh) [inline Fedora_23_Teeth.cfg]: These changes are needed

**Pull request closed without merging**